api/app.ts

```diff
@@ -16,9 +18,11 @@ instance.use(cors({ origin: '*', }));
 instance.use(express.json({ limit: '50mb' }));
 instance.use(express.urlencoded({ extended: true }));
 
-useRouter(instance, '/', article);
-useRouter(instance, '/', category);
+useRouter(instance, '/', article, true);
+useRouter(instance, '/', category, true);
 
+instance.use('/article.create', authRoute);
```

Comment:
Are these two lines still needed? the call to `useRouter` forwards requests though the middleware right?

Reply:
I forgot to delete them. Will fix it!